catalyst/dl/callbacks/metrics/accuracy.py

```diff
@@ -4,13 +4,13 @@
 from catalyst.utils import metrics
 
 
-def _get_default_accuracy_args(num_classes: int) -> List[int]:
+def get_default_accuracy_args(num_classes: int) -> List[int]:
```

Comment:
could you please move it to `catalyst/dl/callbacks/metrics/functional.py` and rename to `get_default_topk_args`?